docs/CONFIGURATION.md

```diff
@@ -57,7 +57,7 @@ release 0.3.0.
 | MONGODB_MAX_NUMBER_OF_RETRIES  | 1     | Meadowlark can be configured to retry on MongoDB failure, for example when two concurrent transactions attempt to access the same resource. By default, Meadowlark will retry once. This can be changed with the MONGODB_MAX_NUMBER_OF_RETRIES environment variable. |
 | MONGO_WRITE_CONCERN        | majority   | See [MongoDB: Write Concern](https://www.mongodb.com/docs/v4.0/reference/write-concern/)                                               |
 | MONGO_READ_CONCERN         | majority   | See [MongoDB: Read Concern](https://www.mongodb.com/docs/v4.0/reference/read-concern/)                                                 |
-| **OPENSEARCH_ENDPOINT**    | (none)     | Only required when the OpenSearch listener is configured for Fastify. Example: "http://localhost:9200"                                 |
+| **OPENSEARCH_ENDPOINT**    | (none)     | Only required when the OpenSearch listener is configured for Fastify. Example: "<http://localhost:9200>"                                 |
```

Comment:
Why make this change? The question also applies below.

Reply:
This was a change made by a [linter rule](https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md034.md), I didn't notice the change, rolling back.